src/trace/listener.ts

```diff
@@ -9,6 +9,8 @@ import { didFunctionColdStart } from "../utils/cold-start";
 import { Source } from "./constants";
 import { patchConsole } from "./patch-console";
 import { SpanContext, TraceOptions, TracerWrapper } from "./tracer-wrapper";
+const datadogLambdaVersion = "3.32.0";
```

Comment:
You could export these are reuse them in the spec file, so you only have to substitute once. Or put them in the constants.ts file. I'd also leave this as a dummy value, eg 'x.xx.x' so if we accidentally push a build with the wrong version we can catch it.

Reply:
Done!